data/EslemOuederni.json

```diff
@@ -0,0 +1,22 @@
+{
+	"name": "Eslem Ouederni",
+	"type": "personal",
+	"bio": "Fullstack developer, I'm looking forward to starting a new journey with open source",
+	"avatar": "https://avatars.githubusercontent.com/u/65737942?v=4",
```

Comment:
The `avatar` field isn't required. Your GitHub avatar will be automatically fetched and updated.
```suggestion
```

Reply:
Thank you for pointing that out!